lib/msal-browser/src/interaction_client/SilentRefreshClient.ts

```diff
@@ -63,6 +63,6 @@ export class SilentRefreshClient extends StandardInteractionClient {
     protected async createRefreshTokenClient(serverTelemetryManager: ServerTelemetryManager, authorityUrl?: string, azureCloudOptions?: AzureCloudOptions): Promise<RefreshTokenClient> {
         // Create auth module.
         const clientConfig = await this.getClientConfiguration(serverTelemetryManager, authorityUrl, azureCloudOptions);
-        return new RefreshTokenClient(clientConfig, this.performanceClient);
+        return new RefreshTokenClient(clientConfig);
```

Comment:
Are we sure this is the right approach? This was an intentional change in the other PR, do the tests need to be updated?

Reply:
Thanks for adding this , have addressed it